cirq/__init__.py

```diff
@@ -168,13 +168,24 @@
     moment_by_moment_schedule,
 )
 
+from cirq.sim import (
+    check_state,
```

Comment:
This method name is ambiguous out of context.

It almost feels better placed as `cirq.testing.assert_state_vector_is_normalized(vector=v, atol=1e-8)`.